docs/notebooks/config.ipynb

```diff
@@ -0,0 +1,94 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## User Settings\n",
+    "\n",
+    "Before starting make sure to define \n",
+    "- the inifiles \n",
+    "- and the path to the bin files, where `jpscore` and `jpsreport` live."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# inifiles\n",
+    "jpscore_inifile = \"files/ini_jpscore.xml\"\n",
+    "jpsreport_inifile = \"files/ini_jpsreport.xml\"\n",
+    "# directory to jpscore and jpsreport binaries\n",
+    "jpscore_dir = \"/Users/chraibi/workspace/jupedsim/jpscore/build/bin\" "
```

Comment:
This path specification has error potential.

Reply:
This is the only place that a user must define these variables.
What do you suggest to eliminate this error potential?

Reply:
I would leave the path empty and give an example as comment.

Reply:
I'll remove the content of `jpscore_dir`, when I'm done with this PR. 
The other files are okay, since the tutorial comes with the `files`

Reply:
Thanks